api/credentials/builtin/init.go

```diff
@@ -2,4 +2,5 @@ package builtin
 
 import (
 	_ "ocm.software/ocm/api/credentials/builtin/github"
+	_ "ocm.software/ocm/api/tech/git/identity"
```

Comment:
I don't feel like this is the right place for this package import.

Reply:
where is the right place? would you mind giving a pointer?